digdag-ui/model.js

```diff
@@ -27,9 +27,9 @@ export type IdName = {
   name: string;
 }
 
-export type NameOptionalId = {
+export type NameId = {
   name: string;
-  id: ?number;
+  id: number;
```

Comment:
True and agreed, however the JS ui handles the id as being always present (see https://github.com/treasure-data/digdag/blob/master/digdag-ui/console.jsx#L299) Should the correct behaviour be checking against the existance of that id js side?